### DISCUSSION TITLE
Header fonts

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#7745] [Q&A]
### Description

In the default setting in quarto, the header (different levels) font is different from the main text font. I have been curious about the rationale here for quite a while. This seems a unique feature of quarto, as in Latex, RMarkdown, Pandoc Markdown, etc. the default setting is always a uniform font. 

I guess my question has two parts:

1. What is the rationale for quarto to make the font of headers different from the font of the main text? I would like to hear and possibly appreciate this rationale.
2. What is the most convenient way to make the font of headers the same as the font of the main text, if I do not appreciate the rationale in question 1?

**Comment by mcanouil** [comment]
Are you talking about a format in particular and which one if that is the case?

**Comment by cscheid** [comment]
It's simple: because we believe it looks better that way. You can change it by [consulting the documentation](https://quarto.org/docs/output-formats/html-basics.html#css-styles)
